lms/djangoapps/edraak_marketing_email/models.py

```diff
@@ -12,7 +12,7 @@ class UnsubscribedUser(models.Model):
     class Meta(object):
         app_label = "edraak_marketing_email"
 
-    user = models.OneToOneField(get_user_model(), db_index=True, on_delete=models.CASCADE)
+    user = models.OneToOneField(User, db_index=True, on_delete=models.CASCADE)
```

Comment:
Any reason you did this change?

Reply:
it was giving an error with `get_user_model`, but I forgot what exactly the error